test/test-fs.c

```diff
@@ -2199,6 +2217,73 @@ TEST_IMPL(fs_utime) {
   return 0;
 }
 
+TEST_IMPL(fs_utime_ex) {
+  utime_check_t checkme;
+  const char* path = "test_file";
```

Comment:
I'd write this as `const char path[] = "test_file"` since you're not reassigning it, likewise on line 2406.